src/board/board.py

```diff
@@ -38,35 +36,74 @@ def render_board(self):
         self.render_background()
 
         # Render pieces except current one
-        for tetromino in self.board_tetrominos:
-            tetromino.render_tetromino()
+        for square in self.board_tetrominos_squares:
+            square.render_square()
 
         # Render the ghost tetromino
         self.ghost_tetromino.render_tetromino()
 
         # Render current playable tetromino
         self.current_tetromino.render_tetromino()
 
+    def get_filled_indices(self):
+        """Returns the number of lines filled"""
```

Comment:
Can you update these new functions to match `pydocstyle` please? See #28 for examples

Reply:
LOL? missing period?